imap_processing/swe/l2/swe_l2.py

```diff
@@ -125,8 +126,14 @@ def calculate_phase_space_density(l1b_dataset: xr.Dataset) -> xr.Dataset:
     # energy in eV value that flux calculation can use.
     phase_space_density_dataset = xr.Dataset(
         {
-            "phase_space_density": (["epoch", "energy", "angle", "cem"], density.data),
-            "energy_in_eV": (["epoch", "energy", "angle"], particle_energy_data),
+            "phase_space_density": (
+                ["epoch", "esa_step", "spin_sector", "cem_id"],
+                density.data,
+            ),
+            "energy_in_eV": (
```

Comment:
Do you want "in_eV" in the variable name, or is that stored in the units / metadata instead?